async_tungstenite::tokio::ConnectStream doesn't implement Debug when feature "tokio-native-tls" is enabled.

Cargo.toml
```
[dependencies]
async-tungstenite = { version = "0.13.0", features = ["tokio-runtime", "tokio-native-tls"] }
```

main.rs
``` rust
use async_tungstenite::tokio::ConnectStream;

#[derive(Debug)]
struct Foo(ConnectStream);

fn main() {
    println!("Hello, world!");
}
```

This produce compile error :    error[E0277]: `async_tungstenite::stream::Stream<TokioAdapter<tokio::net::tcp::stream::TcpStream>, TokioAdapter<tokio_native_tls::TlsStream<tokio::net::tcp::stream::TcpStream>>>` doesn't implement `Debug`

I hope this type implements debug.

--- a/src/stream.rs
+++ b/src/stream.rs
@@ -9,6 +9,7 @@ use std::task::{Context, Poll};
 use futures_io::{AsyncRead, AsyncWrite};
 
 /// Stream, either plain TCP or TLS.
+#[derive(Debug)]
 pub enum Stream<S, T> {
     /// Unencrypted socket stream.
     Plain(S),

